apps/notify/notify_bjs2.js

```diff
@@ -100,7 +100,7 @@ exports.show = function(options) {
   gg.clearRect(x,y, r,b);
   // title bar
   if (options.title || options.src) {
-    gg.setColor(options.titleBgColor||0x39C7).fillRect(x,y, r,y+20);
+    gg.setColor("titleBgColor" in options ? options.titleBgColor : g.theme.dark ? 0x1 : 0x39C7).fillRect(x,y, r,y+20);
```

Comment:
Happy to undo the `titleBgColor in options` here if desired, as a user could pass in `"#000"` to get around the falsy-check

Reply:
No, this looks good to me - thanks!

Personally I'd really like to replace the hard-coded 16 bit color values with `#888`/etc as it's far more maintainable too :)

Reply:
Gotya, sounds good - I could have a zip through and switch them over if you like? To check - in this case it'd be `"#0039C7"` right?

Reply:
Thanks! Nope, you'd have to use:

```
(c=>"#"+[(c&0xF800)>>8,(c&0x7E0)>>3,(c&0x1F)<<3].map(c=>((c+8)>>4).toString(16)).join(""))(0x39C7)
```

So `#444` - the color values are 16 bit RGB565 (why it's such a nightmare to work with them!)